tests/test_mlkit.py

```diff
@@ -351,9 +481,180 @@ def test_model_source_validation_errors(self, model_source):
          ValueError)
     ])
     def test_gcs_tflite_source_validation_errors(self, uri, exc_type):
-        with pytest.raises(exc_type) as err:
+        with pytest.raises(exc_type) as excinfo:
             mlkit.TFLiteGCSModelSource(gcs_tflite_uri=uri)
-        check_error(err.value, exc_type)
+        check_error(excinfo, exc_type)
+
+    def test_wait_for_unlocked_not_locked(self):
+        model = mlkit.Model(display_name="not_locked")
+        model.wait_for_unlocked()
```

Comment:
Perhaps this should return something. False to indicate it didn't wait, True to indicate it waited successfully.

Reply:
If you mean "was it already unlocked when I asked it to wait" then users can just check the .locked property of the model. I'm trying to imagine a scenario where I care if it was already unlocked but don't want to check the .locked property. Can you think of any?
If you mean "we tried to wait but the wait time was exceeded" then it throws exceptions.DeadlineExceeded in that case. To me, the point of this function is to have a model that is unlocked at the end of it.